samples/KK.DotNet.BackgroundTasks.Scheduled.Sample.Web/Pages/Index.cshtml.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace KK.DotNet.BackgroundTasks.Scheduled.Sample.Web.Pages
+{
+    using System.Threading.Tasks;
+    using KK.DotNet.BackgroundTasks.Scheduled.Sample.Web.Backgroundservices;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.RazorPages;
+    using Microsoft.Extensions.Logging;
+
+    public class IndexModel : PageModel
+    {
+        private readonly ILogger<RuntimeSampleTask> runtimeSampleTaskLogger;
```

Comment:
Also matter of preference but: since almost all classes only have one logger, I call them all the same: "logger". I only call them differently if there are different loggers in a given class to make myself and others aware that this might not be the logger one wants to use. Also, the shorter name might allow you to put the parameters in lines 22ff on a single line.

Reply:
That's a good point, I will change that.